data/lib/service/ball_score/ball_score_service.dart

```diff
@@ -182,6 +182,27 @@ class BallScoreService {
       throw AppError.fromError(error, stack);
     }
   }
+
+  Future<int> getPlayerTotalRuns(String matchId, String playerId) async {
```

Comment:
consider to remove it if not used, also every run can not be counted as batsman's run just because he was on strike, let's check if we're considering extras as batsman's runs.